int8 get_status
---
string upd_status